src/api/libcellml/variable.h

```diff
@@ -22,7 +22,7 @@ limitations under the License.
 #include "libcellml/namedentity.h"
 #include "libcellml/types.h"
 
-#ifndef SWIG
+#if defined(_WIN32) && !defined(SWIG)
 template class LIBCELLML_EXPORT std::weak_ptr<libcellml::Variable>;
```

Comment:
What was the final consensus (if any!) on OS stuff? I remember a comment somewhere else about this ... @hsorby do you know?

Reply:
It was about `#undefine`ing things like `TRUE` on some platforms. What we now do is to check if `TRUE` is defined and if it is then we `#undefine` it, which is indeed a much better way of doing it (since it's bullet proof).

Here, the issue is different and I am not sure whether there is much we can do about it.